content/en/platform/corda/4.11/enterprise/node/collaborative-recovery/ledger-recovery/ledger-recovery-flow.md

```diff
@@ -0,0 +1,94 @@
+---
+date: '2023-11-08'
+menu:
+  corda-enterprise-4-11:
+    parent: corda-enterprise-4-11-ledger-recovery
+tags:
+- ledger recovery flow
+- ledger recovery flow parameters
+title: Ledger Recovery flow parameters
+weight: 100
+---
+
+# Ledger Recovery flow parameters
+
```

Comment:
I would usually list parameters and descriptions in a table but maybe this is more consistent with existing C4 content?

Reply:
I think this way is more legible with longer descriptions and warnings in the descriptions. Changed H4 to H2 to be in line with another C4 page listing config fields: https://docs.r3.com/en/platform/corda/4.10/enterprise/node/setup/corda-configuration-fields.html